perfkitbenchmarker/configs/storage_utility.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import numpy
+
+
+class SQLProviderError(Exception):
+  pass
+
+
+class IOPSError(Exception):
+  pass
+
+
+class StorageConfig(object):
+  """Given iops and service provider requirements, return necessary parameters.
+
+  Assumes SSD persistent disks.
+
+  Args:
+    iops: Number of iops required.
+    provider: Provider: 'AWS' or 'GCP'.
```

Comment:
We need to add one more parameter here: storage_type.

For now, you can assuming storage_type is "pd-ssd" for GCP and "ebs-gp2" for AWS. but there are other storage types such as pd-hdd, ebs-piops etc that are not covered by this work which will likely need to be covered in the future.

We need to specifically callout the assumed storage type here so users are aware of the current limitations here.